runtime/plugin/color_cmdline.lua

```diff
@@ -0,0 +1 @@
+require 'vim.ui.cmdline'
```

Comment:
If you do that, you might as well put everything in this file and cut out the middle man...

(Unless this is only temporary and for testing and you'll hook this up properly later.)